SETUP.md

```diff
@@ -71,69 +71,63 @@ Finally, flick the repository's switch to enable CI.
-
-For the next step, you need the [Travis command line client](https://github.com/travis-ci/travis.rb) installed.
-This program is a Ruby gem:
-install it with `gem install travis` (not `apt install travis`, which is a different program).
-After the install, you will need to provide your credentials to login to travis with `travis login`.
+#### Add the public key to GitHub
 
 ```sh
-travis encrypt-file \
-  --com \
-  --repo=$OWNER/$REPO \
-  --force \
-  deploy.key > travis-encrypt-file.log
-```
+# Print the URL for adding the public key to GitHub
+echo "https://github.com/$OWNER/$REPO/settings/keys/new"
 
-`ci/travis-encrypt-file.log` will contain a line such as:
-
-```
-openssl aes-256-cbc -K $encrypted_8cc93e35fb85_key -iv $encrypted_8cc93e35fb85_iv -in test.key.enc -out test.key -d
+# Print the public key for copy-pasting to GitHub
+cat deploy.key.pub
 ```
 
-`ci/deploy.sh` must be updated with the random string from this line (i.e. `8cc93e35fb85`).
-The following command automates this substitution.
+Go to the GitHub settings URL echoed above in a browser, and click "Add deploy key".
+For "Title", add a description like "Manubot Travis Deploy Key".
+Copy-paste the contents of the `deploy.key.pub` text file (printed above by `cat`) into the "Key" text box.
+Check the "Allow write access" box below.
+Finally, click "Add key".
+
+#### Add the private key to Travis CI
 
 ```sh
-# Edit ci/deploy.sh with travis secure env variables generated by travis encrypt-file
-TRAVIS_ENCRYPT_ID=`python -c "import re; \
-  text = open('travis-encrypt-file.log').read(); \
-  print(re.search('encrypted_([a-f0-9]+)_key', text).group(1))"`
-sed "s/9befd6eddffe/$TRAVIS_ENCRYPT_ID/g" deploy.sh > tmp && mv -f tmp deploy.sh
+# Print the URL for adding the private key to Travis CI
+echo "https://travis-ci.com/$OWNER/$REPO/settings"
 ```
 
-Next, limit [concurrent](https://blog.travis-ci.com/2014-07-18-per-repository-concurrency-setting/) Travis CI jobs to ensure previous builds deploy before subsequent ones begin:
+Next, go to the Travis CI repository settings page (URL echoed above).
+Add a new record in the "Environment Variables" section.
+For "NAME", enter `MANUBOT_SSH_PRIVATE_KEY`.
+Next, copy-paste the content of `deploy.key.txt` into "VALUE"
+(including any trailing `=` characters if present).
+Make sure "Display value in build logs" remains toggled off (the default).
 
-```sh
-travis settings \
-  --com \
-  --repo=$OWNER/$REPO \
-  maximum_number_of_builds --set 1
-```
+While in the Travis CI settings, activate the [limit concurrent jobs](https://blog.travis-ci.com/2014-07-18-per-repository-concurrency-setting/) toggle and enter `1` in the value field.
+This ensures previous Manubot builds deploy before subsequent ones begin.
+
+### CI clean up
 
 The continuous integration configuration is now complete.
 Clean up:
 
 ```sh
 # Optionally remove untracked files
 rm deploy.key
-rm travis-encrypt-file.log
```

Comment:
Should we recommend `rm deploy.key.txt` as well?

Reply:
Done in https://github.com/manubot/rootstock/pull/243/commits/470e5a622634700d012b3b626e10c8d658a5446b